aries-core/src/js/components/core/Sidebar/stories/Green.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Button, Box, Grommet, Text, Nav } from 'grommet';
+
+import {
+  Analytics,
+  Chat,
+  Clock,
+  Configure,
+  Help,
+  Projects,
+  StatusInfoSmall,
+} from 'grommet-icons';
+
+import { aries } from '../../../../../../../aries-site/src/themes/aries';
+import { Sidebar } from '../Sidebar';
+
+/* Avatar */
+const SidebarHeader = () => (
+  <Box
+    align="center"
+    round="full"
+    height="xxsmall"
+    width="xxsmall"
+    justify="center"
```

Comment:
This is still getting a little squished on mine. I'd expect it to be a perfect circle:
<img width="70" alt="Screen Shot 2020-03-26 at 10 33 06 AM" src="https://user-images.githubusercontent.com/12522275/77677764-6243d900-6f4d-11ea-9dd1-2829dc2bf2b1.png">

Reply:
My son in response: "squishy things are the best" 
...And sure, I'll take a look :)

Reply:
I believe it's taking care of on my last commit. let me know if it is still funky on your end.